.github/workflows/rust.yml

```
@@ -74,7 +74,7 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly
+          toolchain: stable
```

```suggestion
          toolchain: nightly
```

We can't do that for rustfmt since we're using this `.rustfmt.toml` file:
```toml
unstable_features = true

use_small_heuristics = "max"
imports_granularity = "Module"
group_imports = "StdExternalCrate"
```

The CI seems to work, but it actually doesn't; look at the github action; it outputted one thousand times this warning:
```
Warning: can't set `imports_granularity = Module`, unstable features are only available in nightly channel.
Warning: can't set `group_imports = StdExternalCrate`, unstable features are only available in nightly channel.
Warning: can't set `unstable_features = true`, unstable features are only available in nightly channel.
```

---

Ho you are absolutely right, we should keep nightly for rustfmt. But can't we `-D warnings` for rustfmt too?

---

Nope, this option doesn't exists...